Misc/NEWS.d/next/Library/2025-06-28-11-12-12.gh-issue-136066.WqsKka.rst

```diff
@@ -0,0 +1,2 @@
+:mod:`platform`: eliminate asymptotic quadratic-time complexity
```

Comment:
```suggestion
:mod:`platform`: eliminate asymptotic loglinear complexity
```

Reply:
`re.sub()` has loglinear complexity. Maybe with smaller coefficient, but still. In any case, it does not make much sense to talk about asymptotic complexity when the input is small.

Reply:
> re.sub() has loglinear complexity

Arf. Ok, so what do you suggest as a better message?